src/zcl/zclFrame.ts

```diff
@@ -137,15 +138,16 @@ class ZclFrame {
     /**
      * Parsing
      */
-    public static fromBuffer(clusterID: number, header: ZclHeader, buffer: Buffer): ZclFrame {
+    public static fromBuffer(clusterID: number, header: ZclHeader, buffer: Buffer, customClusters: {[s: string]: ClusterDefinition}): ZclFrame {
         if (!header) {
             throw new Error("Invalid ZclHeader.");
         }
 
         const buffalo = new BuffaloZcl(buffer, header.length);
         const cluster = Utils.getCluster(
             clusterID,
-            header.frameControl.manufacturerSpecific ? header.manufacturerCode : null
+            header.frameControl.manufacturerSpecific ? header.manufacturerCode : null,
```

Comment:
`const cluster = Utils.getCluster(clusterID, header.manufacturerCode, customClusters);`
is enough here, `manufacturerCode` is typed and null automatically if `manufacturerSpecific` was false during parsing.

Reply:
done!